### PR TITLE
Feat/show list task

### PR DESCRIPTION
**Pull request by TIEN1909**
#152 Show task list

**Result**

> Responsive screens

- X-Small devices (portrait phones, from 375px)

image

- Small devices (landscape phones, 576px and up)

image

### CODE REVIEW

File: javascript-training/big-practice/pomodoro-app/src/index.html [hunk 1]
```diff
@@ -22,6 +22,62 @@
     </button>
   </header>
 
+  <main>
+
+    <section class="task container">
+      <div class="heading-task">
+        <p class="title">Tasks</p>
```
Comment (thanhnguyen-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/index.html:29]: why title is a paragraph?

File: javascript-training/big-practice/pomodoro-app/src/styles/components/task.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  background-color: var(--primary-color);
+  cursor: pointer;
+}
+
+.task-item:hover {
+  border-color: #ccc;
+}
+
+.is-click-task {
+  border-color: #222;
```
Comment (thanhnguyen-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/styles/components/task.css:23]: why pure color value here? I expect each color have its own name?

File: javascript-training/big-practice/pomodoro-app/src/styles/pages/home.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  padding-bottom: 14px;
+  margin-bottom: 20px;
+  border-bottom: 2px solid rgba(255, 255, 255, 0.6);
+}
+
+.title {
+  color: var(--primary-text-color);
+  font-size: var(--fs-large);
+  font-weight: var(--fw-extrabold);
+  text-transform: capitalize;
```
Comment (thanhnguyen-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/styles/pages/home.css:14]: I don't see common class css. For example, this line should have class like
```css
.capitalize {
  text-transform: capitalize;
}
```
then use `<tag class="title capitalize">`

File: javascript-training/big-practice/pomodoro-app/src/index.html [hunk 4]
```diff
@@ -22,6 +22,62 @@
     </button>
   </header>
 
+  <main>
+
+    <section class="task container">
+      <div class="heading-task">
+        <h5 class="title capitalize">tasks</h5>
```
Comment (hungdo-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/index.html:29]: Anywhere else you use other heading tags?

File: javascript-training/big-practice/pomodoro-app/src/index.html [hunk 5]
```diff
@@ -22,6 +22,62 @@
...
+        <button class="btn pointer">
+          <img src="./assets/icons/icon-option.png" alt="icon option" class="btn-more">
```
Comment (hungdo-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/index.html:31]: meaningless alt value

File: javascript-training/big-practice/pomodoro-app/src/index.html [hunk 6]
```diff
@@ -22,6 +22,62 @@
...
+          <div class="task-item-left">
+            <span class="icon-checked">
+            </span>
+            <p class="task-name">Task 1</p>
+          </div>
+
+          <div class="task-item-right">
+            <div class="task-est">
+              <span class="actual">2</span>
+              <span>/</span>
```
Comment (hungdo-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/index.html:47]: Why has empty span here?

File: javascript-training/big-practice/pomodoro-app/src/styles/components/button.css [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  padding: 8px;
+  border: none;
+  border-radius: 5px;
+  outline: none;
+  background-color: var(--overlay-color);
+}
+
+.btn-more {
+  width: 16px;
+  height: 16px;
```
Comment (hungdo-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/styles/components/button.css:11]: Why need to fixed width and height?

File: javascript-training/big-practice/pomodoro-app/src/styles/components/task.css [hunk 8]
```diff
@@ -0,0 +1,99 @@
+.task-item,
+.task-item-left,
+.task-item-right {
+  display: flex;
+  align-items: center;
+}
+
+.task-item {
+  justify-content: space-between;
```
Comment (hungdo-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/styles/components/task.css:9]: using without display attribute?

File: javascript-training/big-practice/pomodoro-app/src/styles/components/task.css [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+  overflow-wrap: break-word;
+}
+
+.task-est {
+  margin-right: 18px;
+  color: var(--secondary-text-color);
+  font-weight: var(--fw-extrabold);
+}
+
+.actual {
```
Comment (hungdo-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/styles/components/task.css:74]: What is this class name

Comment (hungdo-agilityio) [reply javascript-training/big-practice/pomodoro-app/src/styles/components/task.css:74]: Class name should be noun :)

File: javascript-training/big-practice/pomodoro-app/src/index.html (hunk 6 above)
Comment (hungdo-agilityio) [reply javascript-training/big-practice/pomodoro-app/src/index.html:47]: My bad I don't notice the splash `/` ignore this comment

File: javascript-training/big-practice/pomodoro-app/src/index.html (hunk 4 above)
Comment (TIEN1909) [reply javascript-training/big-practice/pomodoro-app/src/index.html:29]: no, I only use the heading tag in heading-task

File: javascript-training/big-practice/pomodoro-app/src/styles/components/task.css (hunk 8 above)
Comment (TIEN1909) [reply javascript-training/big-practice/pomodoro-app/src/styles/components/task.css:9]: I use the general display attribute at the top
 
image

File: javascript-training/big-practice/pomodoro-app/src/index.html [hunk 10]
```diff
@@ -22,6 +22,62 @@
...
+            </div>
+
+            <button class="btn btn-edit pointer">
+              <img src="./assets/icons/icon-edit.png" alt="icon edit" class="btn-more icon-edit">
+            </button>
+          </div>
+        </li>
+
+        <li class="task-item">
+          <div class="task-item-left">
```
Comment (hungdo-agilityio) [inline javascript-training/big-practice/pomodoro-app/src/index.html:58]: class name has left, right is not make sense. What if layout change

**Pull request merged**